chapters/planning-data-work.tex

```diff
@@ -1,147 +1,463 @@
-This is done through the creation of globals (in Stata) or string scalars (in R).
-These are ``variables'' -- coding shortcuts that refer to subfolders,
-so that those folders can be referenced without repeatedly writing out their complete filepaths.
-Because the \texttt{/DataWork/} folder is shared by the whole team,
-its structure is the same in each team member's computer.
-What may differ is the path to the project folder (the highest-level shared folder).
-This is reflected in the master script in such a way that
-the only change necessary to run the entire code from a new computer
-is to change the path to the project folder.
-The code in \texttt{stata-master-dofile.do} shows how folder structure is reflected in a master do-file.
+For our group, maintaining a single unified directory structure
+across the entire portfolio of projects means that everyone
+can easily move between projects without having to reorient
+themselves to how files and folders are organized.
+
+Our suggested file structure is not for everyone.
+But if you do not already have a standard file structure across projects,
+it is intended to be an easy template to start from.
+This system operates by creating a \texttt{DataWork} folder at the project level,
+and within that folder, it provides standardized directory structures
+for each data source (in the primary data context, ``rounds'' of data collection).
+For each, \texttt{iefolder} creates folders for raw encrypted data,
+raw deidentified data, cleaned data, final data, outputs, and documentation.
+In parallel, it creates folders for the code files
+that move the data through this progression,
+and for the files that manage final analytical work.
+The command also has some flexibility for the addition of
+folders for non-primary data sources, although this is less well developed.
+The package also includes the \texttt{iegitaddmd} command,
+which can place a \texttt{README.md} file in each of these folders.
+These \textbf{Markdown} files provide an easy and Git-compatible way
+to document the contents of every folder in the structure.
+  \index{Markdown}
+
+The \texttt{DataWork} folder may be created either inside
+an existing project-based folder structure, or it may be created separately.
+It's usually created by the leading RA in agreement with the PI.
+Increasingly, our recommendation is to create the \texttt{DataWork} folder
+separately from the project management materials,
+reserving the ``project folder'' for contracts, Terms of Reference, briefs and other administrative or management work.
+  \index{project folder}
+This is so the project folder can be maintained in a synced location like Dropbox,
+while the code folder can be maintained in a version-controlled location like GitHub.
+(Remember, a version-controlled folder can \textit{never} be stored inside a synced folder,
+because the versioning features are extremely disruptive to others
+when the syncing utility operates on them, and vice versa.)
```

Comment:
For such a string recommendation I think it is a bit imprecise. You can put it in a synced folder as long as you that is not a folder you sync-share with anyone.

What about?

_(Remember, a git folder \textit{should not} be stored inside a folder shared with other
users using a sync service like DropBox, as those two types of collaboration tools works very differently and that difference can lead to strange conflicts if applied at the same time.)_

Reply:
OK 89a538ab1139f5c4871fca0d6bdb87cb34a3ab95